### ISSUE TITLE
Version in JOSS submission does not match GitHub's latest release

### ISSUE THREAD

**Issue by oesteban** [raamana/hiwenet#1]
Here (https://github.com/openjournals/joss-reviews/issues/380#issuecomment-326038826), version 0.2.2 is reported but the repo's version indicates 0.2. The installation script specifies 0.2.2 (https://github.com/raamana/hiwenet/blob/master/setup.py#L6).

I would suggest the versioneer package to keep track automatically of versions.

Just generating the appropriate git tag (0.2.2) would satisfy this comment, though.

**Comment by raamana** [comment]
Thanks Oscar. I'm a little confused here. Are you suggesting me to do this or the JOSS maintainers? :) from what I can see the version number on pypi, on JOSS issue and setup script are all the same.

**Comment by raamana** [comment]
TBH the recent changes have been super minor - like better packaging etc. One this gets published, version numbers will be much more meaningful with fewer releases of more substantial changes.

**Comment by raamana** [comment]
I just checked again. From what I can see, the version number on pypi, on JOSS issue and the setup script in the repo here are all the same. Let me know if I am missing something.

**Comment by oesteban** [comment]
Hi @raamana, I just checked the item in the revision list that says:

> - [ ]  *Version*: Does the release version given match the GitHub release (0.2.2)?

The release version for the submission is 0.2.2 as indicated in that issue, however such version does not exist in your GitHub repo (https://github.com/raamana/hiwenet/releases)

As I said, just  (if your git remotes are standard):

```
git tag 0.2.2
git push origin 0.2.2
```
would suffice to tick the item.

Additionally, I advised not to manually keep track of versions (and let Git do it) by using some software like versioneer. But that is just a hint and hence, not required for acceptance.
